documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc

```diff
@@ -6,53 +6,159 @@
 
 = Enabling the example Grafana dashboards
 
-Set up a Prometheus data source and example dashboards to enable Grafana for monitoring.
+Strimzi provides xref:ref-metrics-config-files-{context}[example dashboard configuration files for Grafana].
+Example dashboards are provided in the `examples/metrics` directory as JSON files:
 
-NOTE: No alert notification rules are defined.
+* `strimzi-kafka.json`
+* `strimzi-zookeeper.json`
+* `strimzi-kafka-connect.json`
+* `strimzi-kafka-mirror-maker-2.json`
+* `strimzi-operators.json`
 
-When accessing a dashboard, you can use the `port-forward` command to forward traffic from the Grafana pod to the host.
+The example dashboards are a good starting point for monitoring key metrics, but they do not represent all available metrics.
+You may need to modify the example dashboards or add other metrics, depending on your infrastructure.
 
-For example, you can access the Grafana user interface by:
+After setting up Prometheus and Grafana, you can visualize the Strimzi data on the Grafana dashboards.
 
-. Running `kubectl port-forward svc/grafana 3000:3000`
-. Pointing a browser to `http://localhost:3000`
+NOTE: No alert notification rules are defined.
+
+When accessing a dashboard, you can use the `port-forward` command to forward traffic from the Grafana pod to the host.
 
 NOTE: The name of the Grafana pod is different for each user.
 
 .Procedure
 
-. Access the Grafana user interface using `admin/admin` credentials.
+. Get the details of the Grafana service:
++
+[source,shell]
+----
+kubectl get service grafana
+----
+
++
+For example:
++
+[table,stripes=none]
+|===
+|NAME     |TYPE      |CLUSTER-IP    |PORT(S)
+
+|grafana  |ClusterIP |172.30.123.40 |80/TCP
+|===
++
+Note the port number for port forwarding.
+
+. Use `port-forward` to redirect the Grafana user interface to `localhost:3000`:
++
+[source,shell]
+----
+kubectl port-forward svc/grafana 3000:80
+----
 +
-On the initial view choose to reset the password.
+NOTE: If using the OpenShift web console, you can switch to menu:Networking[Routes], and click the URL for the `grafana` Route.
```

Comment:
I would probably remove this. There are several other options how to do this on different platforms. Not sure it is worth to pick only one of them. Exposing it using routes is also questionable from security perspective.

Reply:
Okay